dsl/procedures/createIngress/steps/createIngress.groovy

```diff
@@ -0,0 +1,32 @@
+$[/myProject/scripts/preamble]
+
+// Input parameters
+String serviceProjectName = '$[serviceProjectName]'
+String clusterName = '$[clusterName]'
+String envProjectName = '$[envProjectName]'
+// default cluster project name if not explicitly set
+if (!envProjectName) {
+    envProjectName = serviceProjectName
+}
+String environmentName = '$[environmentName]'
+def pluginProjectName = '$[/myProject/projectName]'
+
+def subnetList = '$[subnetList]'
+def gatewayList = '$[gatewayList]'
+def enableIpv6 = '$[enableIpv6]'
+def attachable = '$[attachable]'
+def mtu = '$[mtu]'
+def labels = '$[labels]'
+
+EFClient efClient = new EFClient()
+
+def clusterParameters = efClient.getProvisionClusterParameters(clusterName, envProjectName, environmentName)
```

Comment:
Since this is a regular procedure that will be invoked on demand as part of an application or service process that the user defines, we do not need to go through a service or cluster definition to get the plugin configuration name. Instead simply have a pluginConfig formal parameter in the procedure that the user can use to specify the plugin config name. We can remove the other parameters for service project, env project and the cluster project.